client/view/Form/Form.js

```diff
@@ -147,7 +148,6 @@ regionSelect.appendChild(createOptgroup('Continents', regionGroups.continents))
 regionSelect.appendChild(createOptgroup('Countries', regionGroups.countries))
 
 regionSelect.addEventListener('change', () => {
-  trackEvent('Change region', { props: { region: regionSelect.value } })
```

Comment:
I think it is better to track the user's action, not data usage.

Otherwise one user with many queries will affect our region statistics.

Reply:
👍

I created new Plausible events and updated the title and description of this PR